mne/viz/brain/view.py

```diff
@@ -0,0 +1,280 @@
+              'parietal': View(elev=5, azim=-110),
+              'lat': View(elev=5, azim=0),
+              'med': View(elev=5, azim=180),
+              'fos': View(elev=5, azim=90),
+              'cau': View(elev=5, azim=-90),
+              'dor': View(elev=90, azim=0),
+              'ven': View(elev=-90, azim=0),
+              'fro': View(elev=5, azim=110),
+              'par': View(elev=5, azim=-110)}
+
+
+class TimeViewer(object):
+    u"""Creates time viewer widget.
+
+    Parameters
+    ----------
+    brain : Brain
+        Object with cortex mesh to be controlled by time
+        viewer.
+    """
+
+    def __init__(self, brain):
+        import ipyvolume as ipv
+        import ipywidgets as widgets
+
+        if brain.data['time'] is None:
+            raise ValueError('Brain class instance does not have time data.')
+
+        self._brain = brain
+        time = brain.data['time']
+        time_idx = brain.data['time_idx']
+        overlays = tuple(brain.overlays.values())
+        control = ipv.animation_control(overlays,
+                                        len(time),
+                                        add=False,
+                                        interval=500)
+        slider = control.children[1]
+        slider.readout = False
+        slider.value = time_idx
+        label = widgets.Label(self._get_label(time[time_idx]))
+
+        # hadler for changing of selected time moment
+        def slider_handler(change):
+            k = brain.data['k']
+            b = brain.data['b']
+            lut = brain.data['lut']
+            time_idx_new = int(change.new)
+
+            for v in brain.views:
+                for h in brain.hemis:
+                    act_data = brain.data[h + '_array'][:, time_idx_new]
+                    smooth_mat = brain.data[h + '_smooth_mat']
+                    act_data = smooth_mat.dot(act_data)
+                    act_data = k * act_data + b
+                    act_data = np.clip(act_data, 0, 1)
+                    act_color_new = lut(act_data)
+                    brain.overlays[h + '_' + v].color = act_color_new
+
+            # change label value
+            label.value = self._get_label(time[time_idx_new])
+
+        slider.observe(slider_handler, names='value')
+        control = widgets.HBox((*control.children, label))
+        ipv.gcc().children += (control,)
+
+    def show(self):
+        u"""Display widget."""
+        import ipyvolume as ipv
+        ipv.show()
+
+    def _get_label(self, time):
+        u"""Return time label string.
+
+        Parameters
+        ----------
+        time : float | int
+            Time value to show.
+
+        Returns
+        -------
+        label : str
+            Time label as string.
+        """
+        time_label = self._brain.data['time_label']
+
+        if isinstance(time_label, str):
+            label = time_label % time
+        elif callable(time_label):
+            label = time_label
```

Comment:
I don't get it. The docstring says it returns a str. But this means if time_label is callable, it will return a callable.